src/main/kotlin/Shovel.kt

```diff
@@ -7,45 +7,42 @@ import kotlinx.coroutines.async
 import kotlinx.coroutines.isActive
 import org.apache.kafka.clients.consumer.ConsumerRecord
 import org.apache.kafka.clients.consumer.KafkaConsumer
-import org.slf4j.Logger
-import org.slf4j.LoggerFactory
 import java.io.BufferedInputStream
 import java.text.SimpleDateFormat
 import java.time.Duration
 import java.util.*
 
-val log: Logger = LoggerFactory.getLogger("shovelAsync")
+val log: JsonLoggerWrapper = JsonLoggerWrapper.getLogger("shovelAsync")
 
 fun shovelAsync(kafka: KafkaConsumer<ByteArray, ByteArray>, s3client: AmazonS3, pollTimeout: Duration) =
     GlobalScope.async {
         log.info(Config.Kafka.reportTopicSubscriptionDetails())
         while (isActive) {
-            log.info("Subscribing to '${Config.Kafka.deadLetterQueueTopic}' topic.")
+            log.info("Subscribing", "dead_letter_queue_topic", Config.Kafka.deadLetterQueueTopic)
             kafka.subscribe(setOf(Config.Kafka.deadLetterQueueTopic))
-            log.info("Polling '${Config.Kafka.deadLetterQueueTopic}' topic for '$pollTimeout'.")
+            log.info("Polling", "polling_timeout", "$pollTimeout")
```

Comment:
Could add "queue_name" here

Reply:
It's in the previous log line (the "Subscribing" one